test/ui-test/testSuites/global_shared/scripts/bdd_hooks.py

```diff
@@ -6,6 +6,12 @@
 
 from steps.commonInitSteps import context_init
 
+
+testSettings.waitForObjectTimeout = 5000
```

Comment:
I think we have these propeties already set in `context_init`. Could you check? bc maybe they are rewritten

Reply:
fixed: https://github.com/status-im/status-desktop/pull/10226/commits/995a43034e9a81a8b9833a53ad34f5cd65487bc1